context_test.go

```diff
@@ -0,0 +1,27 @@
+package rpc
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+)
+
+func TestRpcTags(t *testing.T) {
+	logTags := make(CtxRpcTags)
+
+	logTags["hello"] = "world"
+	ctx := AddRpcTagsToContext(context.Background(), logTags)
```

Comment:
can you add another tag to logTags that isn't overwritten by logTags2, and check that it's in outTags? Just to eliminate the possibility that AddRpcTagsToContext overwrites the map instead of merges.